README.md

```diff
@@ -168,20 +168,20 @@ Code snippets like `var foo = "bar";` can be shown inline.
 Also, `this should vertically align` ~~`with this`~~ ~~and this~~.
 
 Code can also be shown in a block element.
-````
+```
```

Comment:
> but I guess that's a style preference.

But all other fenced code blocks in this file have blank lines surrounding them. These 2 are the only inconsistent exceptions.